setup

```diff
@@ -22,7 +22,7 @@ install() {
     . /etc/os-release
     mkdir -Z -p ${rootfs}
     dnf -y install --releasever=${VERSION_ID} --installroot ${rootfs} selinux-policy-targeted podman systemd hirte-agent --nogpgcheck
-    dnf -y update --installroot ${rootfs}
+    dnf -y update --installroot ${rootfs} --nogpgcheck
```

Comment:
Might be worth leaving a FIXME comment here to get rid of --nogpgcheck.

Reply:
Yes I need to talk to Alex about this to see if he has any ideas or potentially some installation people.